src/main/java/com/rocketden/main/dto/room/CreateRoomResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.rocketden.main.dto.room;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateRoomResponse {
+    public static final String SUCCESS = "Successfully created room.";
```

Comment:
Do you think we should also include some sort of `CANNOT_CREATE_ROOM` object? Or maybe we only include that later when that error could predictably arise (like when the host is already in another room or something). I'm fine with either, especially since I mentioned earlier to not include additions that are not used in the PR.

Reply:
I was thinking that too but then decided against it for exactly that reason you mentioned. In the future though I think we'll definitely need to have that.